src/logic/errorDispatch.ts

```diff
@@ -1,10 +1,12 @@
 // IMPORTANT: this should match 1:1 with the MutinyJsError enum in mutiny-wasm
 // If we can handle all of these, we can handle all the errors that Mutiny can throw
 
-// WARNING: autogenerated code, generated by errorsToTs.cjs
+// WARNING: autogenerated code, generated by calling:
+// `node scripts/errorsToTs.cjs /path/to/mutiny-wasm/src/error.rs`
```

Comment:
Is there a way we could have CI check that this is up-to-date?

Reply:
ideally this would be generated by wasm-bindgen (something like a string enum: https://github.com/rustwasm/wasm-bindgen/issues/3057) ... anything else is gonna be hacky